### DISCUSSION TITLE
Hack Yi Home 1080 9FUS with firmware 12.2.0

### DISCUSSION THREAD

**Discussion by nkieto** [roleoroleo/yi-hack-Allwinner#387] [Q&A] [answered]
I have seen that is possible to hack 9FUS (yi20ga) camera if you have 8.2.0 firmware.

Is it possible to downgrade my 12.2.0 firmware to 8.2.0? Do you know where can I download this old firmare?

Or can I try to hack directly with firmware 12.2.0 installed?

Thanks in advanced with any help

**Comment by Giuserver** [comment, marked as answer]
Here: https://www.kamihome.com/firmware/

**Comment by roleoroleo** [comment]
You can downgrade the firmware and hack v8.
Or wait for the next release that will be compatible with v12.
